underlay/src/main/resources/config/underlay/sd/underlay.json

```diff
@@ -75,7 +75,7 @@
 
     "omop/notePerson",
 
-    "omop/cpt4Person",
+    "sd/cpt4Person",
```

Comment:
Did this change at all? If not, you can keep using the `omop` version, no need to duplicate, even if the entities referenced in the group changed.

Reply:
since I change the cpt4, i preferred to find the cptePerson in the same sd/ tree. Layter when standardizing I am planning to make a copy of **all**  omop -> so that we would not have to update if you do change any omop tables going forward.

Reply:
I see, same as you did for the AoU. OK makes sense.